recipes/com/github/kevinstern/scm.yaml

```diff
@@ -0,0 +1,6 @@
+---
+type: "git"
+uri: "https://github.com/KevinStern/software-and-algorithms.git"
+tagMapping:
+  - pattern: 1.0
+    tag: d428bbb
```

Comment:
Does a "tag" that's a commit hash work? I thought that I saw it fail when used in some existing scm yaml file already.

Reply:
It should, if it doesn't then it is a bug and we should fix it.